### DISCUSSION TITLE
Feature Request: queries in Y-Max / Y-Min

### DISCUSSION THREAD

**Discussion by minfrin** [grafana/grafana#38379] [Dashboards, Panels & Visualizations]
**What would you like to be added**:

You have a panel containing disks on a server, and how full they are. You want to graph the disk used out of the total disk. Both "disk used" and "total disk" come from queries. The Y-Min and Y-Max in this case come from the query.

Most specifically, I want query A to be the value, query B to the Y-Min, and query C to be Y-Max.

This is a different approach to https://github.com/grafana/grafana/issues/3935.

**Why is this needed**:

Right now it is impossible to graph disk space, or anything else represented as a fraction.

You end up with this meaningless graph:

<img width="746" alt="grafana-meaningless-graph" src="https://user-images.githubusercontent.com/1331788/82381008-c2289e80-9a29-11ea-94d9-caf708a20438.png">

All widths are calculated as a percentage of the biggest width, which is arbitrary and has no relation to the others.

**Comment by minfrin** [comment]
To summarise, right now this graph is impossible to render in grafana:

![image](https://user-images.githubusercontent.com/1331788/82383887-05d1d700-9a2f-11ea-93b9-81b1be0ecc1b.png)

https://collectd.org/wiki/index.php/Plugin:DF

**Comment by pkolyvas** [comment]
We're in the process of moving Grafana feature requests to [GitHub Discussions](https://community.grafana.com/t/were-moving-grafana-feature-requests-to-github-discussions/50341/3). This issue is being moved to [Discussions](https://github.com/grafana/grafana/discussions) or has been moved from our issues list, depending on where you're seeing this message. 

 We hope to use the discussion thread to continue building use cases, and additional evidence based on engagement/voting, in order to help us prioritize this feature request later on down the road. 

As with all things Grafana, we value your input and participation in this project and appreciate your patience during this transitional period. Looking forward to the discussion!

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
